docs/Installation/topics/Installing_Zenko.rst

```diff
@@ -0,0 +1,230 @@
+Installing Zenko
+################
+
+Set up a cluster of five nodes conforming to the specifications listed above.
+If you are using MetalK8s, do this by downloading the latest stable MetalK8s
+source code from the MetalK8s releases page:
+https://github.com/scality/metalk8s/releases. Follow the Quickstart guide
+(in docs/usage/quickstart.rst) to install MetalK8s on your cluster.
+
+When building your cluster, take sizing into account. If you are deploying
+non-default sizing, make sure your volume sizing is sufficient. For MetalK8s,
+you *must* size the volumes in the inventory during setup in
+metalk8s/inventory/group_vars/kube-node.yml.
+
+For a default sizing, paste the following into kube-node.yml:
+
+.. code-block:: yaml
+		
+  metalk8s_lvm_default_vg: False
+  metalk8s_lvm_vgs: ['vg_metalk8s']
+  metalk8s_lvm_drives_vg_metalk8s: ['/dev/vdb']
+  metalk8s_lvm_lvs_vg_metalk8s:
+    lv01:
+      size: 125G
+    lv02:
+      size: 125G
+    lv03:
+      size: 125G
+    lv04:
+      size: 62G
+    lv05:
+      size: 62G
+
+For custom sizing, increase these base numbers.
+
+For non-MetalK8s deployments, follow your vendor or community’s instructions for
+configuring persistent voloumes at 500 Gi/node.
+
+
+.. note::
+
+   It is a best practice to install Zenko on a fresh cluster.
+
+Get Ready
+*********
+
+1. If you are using MetalK8s, use the MetalK8s virtual shell. Change to the
+   directory from which you will deploy Zenko:
+   ::
+
+    (metal-k8s) $ cd /path/to/installation
+
+   If you are not installing from MetalK8s, follow the instructions in
+   Zenko/docs/gke.md to install Helm on your cluster.
+
+2. Initialize Helm:
+   ::
+
+    (metal-k8s) $ helm init
+    Creating /home/centos/.helm
+    Creating /home/centos/.helm/repository
+    Creating /home/centos/.helm/repository/cache
+    Creating /home/centos/.helm/repository/local
+    Creating /home/centos/.helm/plugins
+    Creating /home/centos/.helm/starters
+    Creating /home/centos/.helm/cache/archive
+    Creating /home/centos/.helm/repository/repositories.yaml
+    Adding stable repo with URL: https://kubernetes-charts.storage.googleapis.com
+    Adding local repo with URL: http://127.0.0.1:8879/charts
+    $HELM_HOME has been configured at /home/centos/.helm.
+    Warning: Tiller is already installed in the cluster.
+    (Use --client-only to suppress this message, or --upgrade to upgrade Tiller to the current version.)
+    Happy Helming!
+    (metal-k8s) $
+
+   Helm can now install applications on the Kubernetes cluster.
+
+3. Go to https://github.com/Scality/Zenko/releases and download the latest
+    stable version of Zenko.
```

Comment:
alignment error here is causing the page to render an extra indent.

Reply:
fixed